manim/animation/animation.py

```diff
@@ -248,6 +248,7 @@ def __init__(
         self.duration = duration
         self.mobject = None
         self.stop_condition = stop_condition
+        self.is_static_wait = False
```

Comment:
Though I agree with you, I chose this name because it is how this type of wait is called within the codenbase. 

I don't really like has_udapters, because it's not really what this variable is supposed to express here. 
I thinl `is_frozen_wait` is fine, what do you think ?